x-pack/plugins/cases/common/api/cases/case.ts

```diff
@@ -174,6 +183,10 @@ export const CasesFindRequestRt = rt.partial({
    * The severity of the case
    */
   severity: CaseSeverityRt,
+  /**
+   * The uids of the user profiles to filter by
+   */
+  assignees: rt.union([rt.array(rt.string), rt.string]),
```

Comment:
This needs to be a `string | string[]` just like `tags` because of the way arrays are serialized. This is handled by the `constructQuery` in the find empty.